telematic_system/docker-compose.local.yml

```diff
@@ -101,8 +101,23 @@ services:
         - DB_USER=telematic
         - DB_PASSWORD=telematic
         - GRAFANA_DB=wfd_grafana
+        # File upload service settings
+        - UPLOAD_DESTINATION=s3
```

Comment:
What happens to this upload destination variable for local deployment?
Should this be a bool instead?

Reply:
If that is the only purpose, i think having a bool for is_cloud_deployment will allow for more flexibility in how variable can be used

Reply:
I would prefer to use string with more descriptive name saying which service in cloud is used for storing files.

Reply:
I guess i'm still not very clear on how this is supposed to be used and what is the purpose of this variable.
1. Is s3 and host the only possible values this can take?
2. Is this just optional metadata?
3. Can the user choose where to upload the rosbag? Or is that static for each deployment of the web ui?

Reply:
- If the variable equals to "s3", it will upload the files to S3 bucket. Otherwise, it will be in local folder. "host" is just an example in the JSON. We can support other values if we have other upload destinations.

- This is a mandatory metadata.

- I am not sure it is part of the requirements that we should allow users to choose where to upload files through UI. The upload destination is set during the deployment. If there is a requirement change, we can discuss further.

Reply:
@adev4a Addressed